components/charts/AnnualizedHousingChart.tsx

```diff
@@ -40,8 +44,6 @@ interface AnnualizedHousingChartProps {
   showTrend?: boolean;
 }
 
-
-
```

Comment:
No functional changes in this file. I only made a small change to this file and tested the automated formatting pre-commit.